### PR TITLE
Add missing ObjectType import for notify-only properties

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: src/analysis/properties.rs [hunk 1]
```diff
@@ -226,7 +226,7 @@ fn analyze_property(
         None
     };
 
-    if !generate_trait && (writable || readable) {
+    if !generate_trait && (writable || readable || notifable) {
```
Comment (GuillaumeGomez) [inline src/analysis/properties.rs]: "notifable" -> Not very english haha

Comment (sdroege) [reply src/analysis/properties.rs]: Yeah, missing a letter :) Should I fix in this PR or some other time?

Comment (sdroege) [reply src/analysis/properties.rs]: Ah just did

Comment (GuillaumeGomez) [reply src/analysis/properties.rs]: Hehe.

**Comment by GuillaumeGomez** [comment]
👍

**Comment by EPashkin** [comment]
@sdroege Thanks

**Pull request merged**